### PR TITLE
add list of must watch bollywood movies

### PR DESCRIPTION
**Pull request by nrngchetan**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2531)

### CODE REVIEW

File: hindi/bollywood_mustwatch_movies.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+A list of some must watch bollywood movies (and their IMDB links):
```
Comment (casey-collab) [inline hindi/bollywood_mustwatch_movies.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline hindi/bollywood_mustwatch_movies.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by nrngchetan** [comment]
Please review this @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@nrngchetan, is this list of movies related to the rest of the story? (I don't speak Hindi, so I'm not sure.) The purpose of this repository is to continue the existing story rather than add something completely new.

By the way, I think Casey is wrong about the merge conflict.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**